sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -965,14 +965,20 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   }
 
   test("sanity test for SPARK-6618") {
-    (1 to 100).par.map { i =>
```

Comment:
this test was failing and complained that the execution id is already set. I don't know the detail about how we schedule the 100 tasks on threads with `par`, so I changed it to use `Thread` directly to be more clear. I'll reduce the number of threads.